src/e_uadk.c

```diff
@@ -60,33 +65,42 @@ static int uadk_finish(ENGINE *e)
 static int bind_fn(ENGINE *e, const char *id)
 {
 	struct uacce_dev_list *list;
+	const char *acc_device = "hisi_acc";
+	static int loaded;
 
 	if (id && (strcmp(id, engine_uadk_id) != 0)) {
 		fprintf(stderr, "wrong engine id\n");
 		return 0;
 	}
 
+	if (loaded) {
+		fprintf(stderr, "\n%s\n", __func__);
+		return 0;
+	}
+	loaded = 1;
+
```

Comment:
why need this?

Reply:
> no rsa.c?
> change commit message, remove history info 1,2,3...
> add signed off: wangzhou?

I've pushed the newest uadk_rsa.c in "dev2" branch yesterday, but seems that it still has problems, I'll fix it. 
Sorry, I forgot to change the signed off after merging history commits and using the "git commit -s -m ", I'll add the signed off : Zhou Wang.

Reply:
> why need this?

The definition and assignment of variate "load" seem to be  redundant code, I'll clear it.